kubernetes-model-generator/openshift-model-machineconfiguration/src/generated/java/io/fabric8/openshift/api/model/machineconfiguration/v1/ControllerConfigStatusConditions.java

```diff
@@ -55,11 +55,12 @@
     @BuildableReference(PersistentVolumeClaim.class)
 })
 @Generated("jsonschema2pojo")
-public class ControllerConfigStatusCondition implements Editable<ControllerConfigStatusConditionBuilder> , KubernetesResource
+public class ControllerConfigStatusConditions implements Editable<ControllerConfigStatusConditionsBuilder> , KubernetesResource
```

Comment:
Do you think this renaming is intentional? I'm slightly confused by the name (it's in plural form).

Reply:
This is due to the way the class names are generated, there is no intention in the naming.
If no shared component is defined, class names are just inferred from the OpenAPI inline object forms.

This class is probably used in the `ControllerConfigStatus` class in a field name `conditions` which is likely a list but there can't be certainty about it just from the name.
So in this case, the name of the class is the concatenation of `ControllerConfigStatus` plus the capitalized version of conditions -> `Conditions`.